prime-angular-frontend/src/app/modules/paper-enrolment/pages/overview-page/overview-page.component.html

```diff
@@ -13,254 +13,17 @@
-    </app-page-subheader>
-
-    <ng-container *ngFor="let careSetting of enrollee.enrolleeCareSettings">
-
-      <ng-container *ngFor="let oboSite of enrollee.oboSites">
-        <ng-container *ngIf="oboSite.careSettingCode === careSetting.careSettingCode">
-          <app-enrollee-property title="Care Setting"
-                                 [makeBold]="true">
-            <div class="mb-3">{{ careSetting.careSettingCode | configCode: 'careSettings' }}
-              <span *ngIf="oboSite.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY">
-                ({{ oboSite.healthAuthorityCode | configCode: 'healthAuthorities' | capitalize: true }})
-              </span>
-            </div>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Site Name"
-                                   [makeBold]="true">
-              {{ oboSite.siteName | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Facility Name"
-                                   [makeBold]="true">
-              {{ oboSite.facilityName | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Site ID"
-                                   [makeBold]="true">
-              {{ oboSite.pec | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property title="Job Title"
-                                   [makeBold]="true">
-              {{ oboSite.jobTitle | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property title="Site Address"
-                                   [makeBold]="true">
-              <app-enrollee-property title="Street">
-                {{ oboSite.physicalAddress?.street | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="City">
-                {{ oboSite.physicalAddress?.city | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="Province">
-                {{ oboSite.physicalAddress?.provinceCode | configCode: 'provinces' | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="Postal Code">
-                {{ oboSite.physicalAddress?.postal | postal | default }}
-              </app-enrollee-property>
-
-            </app-enrollee-property>
-          </app-enrollee-property>
-        </ng-container>
-      </ng-container>
-    </ng-container>
-
-  </app-page-section>
+  <app-obo-sites-overview *ngIf="enrollee?.oboSites?.length"
+                          [oboSiteForm]="enrollee">
+  </app-obo-sites-overview>
 
   <app-enrollee-self-declarations *ngIf="enrollee?.selfDeclarations"
```

Comment:
You'll still want to make a component for this page even though it's a single component already so we're consistent where the overview component is located